packages/beacon-node/test/perf/network/gossip/snappy.test.ts

```diff
@@ -0,0 +1,61 @@
+import {randomBytes} from "node:crypto";
+import * as snappyjs from "snappyjs";
+import * as snappy from "snappy";
+import {itBench} from "@dapplion/benchmark";
+
+describe("network / gossip / snappy", () => {
+  const msgLens = [100, 200, 300, 400, 500, 1000, 10000, 100000];
```

Comment:
I used to see blocks with 2MB, need to add 2MB, 5MB and 10MB (max size of gossip message) message length